packages/memory-storage/src/resource-clients/request-queue.ts

```diff
@@ -163,6 +175,98 @@ export class RequestQueueClient extends BaseClient implements storage.RequestQue
         };
     }
 
+    async listAndLockHead(options: storage.ListAndLockOptions): Promise<storage.ListAndLockHeadResult> {
```

Comment:
This (and the prolong/delete lock) method need to also be added to `@crawlee/storage`

Reply:
Maybe it's the impostor syndrome showing, but... what is `@crawlee/storage`? 😅 We definitely need to add support to [`local storage`](https://github.com/apify/apify-storage-local-js), is that it, or there is another place?

Reply:
Wooops. I meant in the client interfaces in `@crawlee/types`, my bad 😅😅

Reply:
Right, that also makes sense 😄 will do